airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1615,6 +1639,20 @@ components:
             - days
             - weeks
             - months
+    # LOGS
+    LogSource:
+      type: string
+      description: the application that generated the logs
+      enum:
+        - server
+        - scheduler
+    LogsRequestBody:
+      type: object
+      required:
+        - source
+      properties:
+        source:
```

Comment:
i might advocate for logType instead of source since source is already a bit overloaded. but up to you.